io/aligned_buf_pool.cc

```diff
@@ -0,0 +1,99 @@
+#include "io/aligned_buf_pool.h"
+
+#include "base/logging.h"
+#include "io/io.h"
+#include "util/fibers/proactor_base.h"
+#include "util/fibers/uring_proactor.h"
+
+namespace io {
+
+namespace {
+
+thread_local AlignedBufPool tl_buf_pool;
+
+}  // namespace
+
+void AlignedBufPool::Init(size_t backing_size) {
```

Comment:
I think it's a bit mixed with both OS stuff and data structure stuff. 
if we have a separate class, that acts as an allocator (resource manager) why not make it independent from the os stuff?

i.e. Init provides a bank of memory registered and allocated somewhere and the class does not really care about iouring.

Reply:
and it you make it completely abstract, it can reside in helio/base actually.